babel.config.js

```diff
@@ -14,6 +14,23 @@ const defaultPlugins = [
 
     // Keep it last
     'react-native-reanimated/plugin',
+    [
+        'module-resolver',
+        {
+            alias: {
+                '@components': './src/components',
+                '@action': './src/libs/actions',
```

Comment:
```suggestion
                '@actions': './src/libs/actions',
```

Reply:
As we use `@actions/core` package for custom github actions this raising conflicts while running linter. So adding `@actions` won't work. Need to check for some alternatives.

Reply:
Maybe `@userActions`